ClippedReLU.lua

```diff
@@ -0,0 +1,13 @@
+local ClippedReLU, parent = torch.class('cudnn.ClippedReLU','cudnn._Pointwise')
+
+function ClippedReLU:__init(ceiling, inplace)
+    parent.__init(self)
+    assert(ceiling, "No ceiling was given to ClippedReLU")
+    self.ceiling = ceiling
+    self.inplace = inplace or false
+end
+
+function ClippedReLU:updateOutput(input)
+    if not self.mode then self.mode = 'CUDNN_ACTIVATION_CLIPPED_RELU' end
```

Comment:
I think it's better to set mode like in BatchNormalization.lua, but that's minor